webplugin/js/app/kommunicate.js

```diff
@@ -931,4 +931,26 @@ $applozic.extend(true, Kommunicate, {
             }
         }
     },
+    getCurrentPosition:() => 
+        new Promise((resolve, reject) => 
+          navigator.geolocation.getCurrentPosition(resolve, reject)
+        ),
+    getUserLocation: async function (){
```

Comment:
@anujpanwar01 can we use async and Promise in chat widget?

Reply:
yes we can use all JS latest features I have added the babel already in the widget code which will convert that code to ie11 compatible